examples/time_series/noaa_json.py

```diff
@@ -0,0 +1,72 @@
+"""
+===================================================================
+Plotting of near real time (NRT) data by fetching it from json file
+====================================================================
+
+How to query and visualize nrt data(json format) using pandas and plot 
+it using TimeSeries Library
+"""
+
+
+
+import pandas as pd 
+from sunpy.time import parse_time
+from sunpy import timeseries as ts 
+from collections import OrderedDict
+from astropy import units as u 
+
+data = pd.read_json("https://services.swpc.noaa.gov/json/goes/primary/xrays-7-day.json")
+
+#################### retrieval ########################
```

Comment:
These are not how the comment blocks are created in examples.

Reply:
You should look over the other examples and see how they are done and replicate that.

Reply:
We use https://sphinx-gallery.github.io/stable/index.html to create these examples and its important to follow their syntax for that.